src/code/code_8012EC80.c

```diff
@@ -98,21 +98,21 @@ u32 gGsFlagsShift[] = {
 };
 
 TexturePtr gItemIcons[] = {
-    gItemIconOcarinaOfTimeTex,       // ITEM_OCARINA
+    gItemIconOcarinaOfTimeTex,       // ITEM_OCARINA_OF_TIME
     gItemIconBowTex,                 // ITEM_BOW
     gItemIconFireArrowTex,           // ITEM_ARROW_FIRE
     gItemIconIceArrowTex,            // ITEM_ARROW_ICE
     gItemIconLightArrowTex,          // ITEM_ARROW_LIGHT
     gItemIconFairyOcarinaTex,        // ITEM_OCARINA_FAIRY
     gItemIconBombTex,                // ITEM_BOMB
     gItemIconBombchuTex,             // ITEM_BOMBCHU
-    gItemIconDekuStickTex,           // ITEM_STICK
-    gItemIconDekuNutTex,             // ITEM_NUT
+    gItemIconDekuStickTex,           // ITEM_DEKU_STICK
+    gItemIconDekuNutTex,             // ITEM_DEKU_NUT
     gItemIconMagicBeansTex,          // ITEM_MAGIC_BEANS
     gItemIconSlingshotTex,           // ITEM_SLINGSHOT
     gItemIconPowderKegTex,           // ITEM_POWDER_KEG
-    gItemIconPictoBoxTex,            // ITEM_PICTO_BOX
-    gItemIconLensofTruthTex,         // ITEM_LENS
+    gItemIconPictoBoxTex,            // ITEM_PICTOGRAPH_BOX
```

Comment:
Did this need to change to `gItemIconPictographBox`?

Reply:
Bump, should probably be `PictographBox` for consistency.